src/commands/Fun/mypets.js

```diff
@@ -22,6 +23,6 @@ module.exports = class extends Command {
 			return formatted.push(`${pet.emoji} ${pet.name}: ${userPets[id]}`);
 		});
 
-		return msg.send(formatted.join('\n'));
+		return queuedMessageSend(this.client, msg.channel.id, formatted.join('\n'));
```

Comment:
whats the point of doing this?

Reply:
mylife has too many pets so it errors when trying to send the message without splitting

Reply:
i think it would be better to do just `, { split: true }` here, `queuedMessageSend` is used only for the minion tasks finishing as an extra precaution to not overstep ratelimits, hugely so after a reboot when 500-1500 trips could be finishing at once.